packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -45,6 +45,9 @@ export const messages = {
   force_auth_web_login_authorize_dev_hub_text: 'SFDX: Authorize a Dev Hub',
   force_auth_web_login_authorize_org_text: 'SFDX: Authorize an Org',
 
+  parameter_directory_strict_not_available:
+    'A required metadata folder named "%s" does not exist in this workspace.',
```

Comment:
@ruthemmanuelle Let's review this message. It is not consistent with the CLI.

Reply:
@ntotten - Do you happen to know what message the CLI uses?

Reply:
@ruthemmanuelle `▸    !   Lightning bundles must have a parent folder named 'lightningcomponents'.`

Reply:
Thanks, @ntotten . @muenzpraeger , is there any reason we can't just use that message?

Reply:
@muenzpraeger was saying that this message is more generic and used for other commands so it may actually be okay now, and shouldnt be the same as the CLI.